packages/core/template/project/DeveloperGuide.md

```diff
@@ -0,0 +1,16 @@
+---
+  layout: default.md
+  title: "Developer Guide"
+  pageNav: 3
+---
+
+# ProjectEx Developer Guide
```

Comment:
I think this page can have a bit more content to differentiate it from the User Guide and be a better jumping off place for users